force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -271,11 +354,19 @@ public with sharing class FlowControlLoopsCollections {
      */
     public static Integer findWordInList(String wordToFind, List<String> words) {
         // The variable to store the index of the word
+        Integer wordLocation = -1;
 
         // Loop through the list of words
+        for (Integer x = 0; x < words.size(); x++) {
+
             // If the current word is the word to find, exit the loop
+            if (words[x] == wordToFind) {
+                wordLocation = x;
+                break;
+            }
+        }
 
-        return null; // Replace null with the variable you used to store the result
+        return wordLocation; // Replace null with the variable you used to store the result
```

Comment:
Good solution here. To be a little more concise I would just  return my index from my for loop the when the condition is true. 

```
 public static Integer findWordInList(String wordToFind, List<String> words) {
     for(Integer i = 0; i < words.size(); i++){
         if(words[i] == wordToFind){
             return i;
         }
     }
        return -1;
    }
 ```